contracts/staking/Middleware.sol

```diff
@@ -0,0 +1,229 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/ContextUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/introspection/ERC165Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/extensions/AccessControlEnumerableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+
+import "../interfaces/staking/symbiotic/IVetoSlasher.sol";
+import "../interfaces/staking/symbiotic/IInstantSlasher.sol";
+import "../interfaces/staking/symbiotic/IVault.sol";
+import "../periphery/interfaces/IAttestationVerifier.sol";
+
+contract AttestationAutherSample is Initializable,  // initializer
+    ContextUpgradeable,  // _msgSender, _msgData
+    ERC165Upgradeable,  // supportsInterface
```

Comment:
I think unnecessary to inhearit `Initializable`, `ContextUpgradeable`, `ERC165Upgradeable` as `AccessControlUpgradeable` inherits them all

Reply:
Yup, we are just trying to list all dependencies here.